ResidentInformationApi.Tests/V1/UseCase/ListContactUseCaseTests.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using ResidentInformationApi.V1.Boundary.Requests;
+using ResidentInformationApi.V1.Boundary.Responses;
+using ResidentInformationApi.V1.Factories;
+using ResidentInformationApi.V1.Gateways;
+using ResidentInformationApi.V1.UseCase;
+using AcademyClaimantInformation = ResidentInformationApi.V1.Domain.AcademyClaimantInformation;
+using HousingResidentInformation = ResidentInformationApi.V1.Domain.HousingResidentInformation;
+using MosaicResidentInformation = ResidentInformationApi.V1.Domain.MosaicResidentInformation;
+
+namespace ResidentInformationApi.Tests.V1.UseCase
+{
+    [TestFixture]
+    public class ListContactUseCaseTests
+    {
+        private Mock<IAcademyInformationGateway> _mockAcademyGateway;
+        private Mock<IHousingInformationGateway> _mockHousingGateway;
+        private Mock<IMosaicInformationGateway> _mockMosaicGateway;
+        private ListContactsUseCase _classUnderTest;
+        private Uri _academyUrl;
+        private Uri _housingUrl;
+        private Uri _mosaicUrl;
+        private readonly Fixture _fixture = new Fixture();
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockAcademyGateway = new Mock<IAcademyInformationGateway>();
+            _mockHousingGateway = new Mock<IHousingInformationGateway>();
+            _mockMosaicGateway = new Mock<IMosaicInformationGateway>();
+            _classUnderTest = new ListContactsUseCase(
+                _mockAcademyGateway.Object,
+                _mockHousingGateway.Object,
+                _mockMosaicGateway.Object
+                );
+            _academyUrl = new Uri("http://ACADEMYAPI_URL");
+            _housingUrl = new Uri("http://HOUSINGAPI_URL");
+            _mosaicUrl = new Uri("http://MOSAICAPI_URL");
+
+            Environment.SetEnvironmentVariable("ACADEMYAPI_URL", _academyUrl.OriginalString);
+            Environment.SetEnvironmentVariable("HOUSINGAPI_URL", _housingUrl.OriginalString);
+            Environment.SetEnvironmentVariable("MOSAICAPI_URL", _mosaicUrl.OriginalString);
+        }
+
+        [Test]
+        public async Task ExecuteReturnsContactInformationResponse()
+        {
+            var stubbedAcademyClaimant = _fixture.Create<AcademyClaimantInformation>();
+            var stubbedHousingResident = _fixture.Create<HousingResidentInformation>();
+            var stubbedMosaicResident = _fixture.Create<MosaicResidentInformation>();
+
+            var academyUrl = new Uri(Environment.GetEnvironmentVariable("ACADEMYAPI_URL"));
+            var housingUrl = new Uri(Environment.GetEnvironmentVariable("HOUSINGAPI_URL"));
+            var mosaicUrl = new Uri(Environment.GetEnvironmentVariable("MOSAICAPI_URL"));
+
```

Comment:
There is some repetition going on here. you have `_academyUrl` as a URI object, you then extract a string from that to set a new environment variable, and then create a new Uri object here. I think you can use the variables you create in the SetUp here.